openfecwebapp/templates/advanced.html

```diff
@@ -84,18 +82,13 @@
           </div>
         </div>
         <div class="option__aside">
-          <aside class="card card--primary is-disabled">
-            <span class="card__image card__icon i-committee"><span class="u-visually-hidden">Icon representing a committee</span></span>
-            <div class="card__content">
-              PAC interactive overviews
-            </div>
-          </aside>
-          <aside class="card card--primary card--stacked is-disabled">
-            <span class="card__image card__icon i-committee"><span class="u-visually-hidden">Icon representing a committee</span></span>
-            <div class="card__content">
-              Party committee interactive overviews
-            </div>
-          </aside>
+          <i class="icon-circle--committee is-disabled"><span class="u-visually-hidden">Icon of a committee</span></i>
+            <a href="#" class="is-disabled">PAC data overview</a><br><br>
+            <a href="#" class="is-disabled">Party committee data overview</a>
+          <i class="icon-circle--checklist card--stacked"><span class="u-visually-hidden">Icon of a checklist</span></i>
```

Comment:
Is it ok to use `card--stacked` class here? That's the way I found how to create space between the icon items